g_docs.go

```diff
@@ -749,49 +757,56 @@ func (res *objectResource) parse() {
 		ColorLog("Unknown type without TypeSec: %v\n", res.object)
 		os.Exit(1)
 	}
-	// TODO support other types, such as `ArrayType`, `MapType`, `InterfaceType` etc...
```

Comment:
Are you sure to remove this TODO?

Reply:
oh good catch! These types are currently not used in DOR. I wanted to add a log in the `default:` section of the switch case, so we can address it right away when we really need to use these types. 

Will add some logs in the `default:`

Reply:
Addressed in https://github.com/zalora/bee/pull/18/commits/8d8c52a78db734e78368178ee617124bd85988a3